docs/oxAuth_swagger.yaml

```diff
@@ -0,0 +1,2030 @@
+# raw swagger spec link:
+# https://raw.githubusercontent.com/GluuFederation/oxAuth/version_4.0/oxAuth-server/src/main/resources/swagger.yaml
+# https://gluu.org/swagger-ui/?url=https://raw.githubusercontent.com/GluuFederation/oxAuth-server/version_4.0/oxAuth-server/src/main/resources/swagger.yaml#/developers/setup-client
```

Comment:
correct this link according to above correction

Reply:
Rectified this link